### PR TITLE
MES-4670, MES-4509 - feat(SafetyAndBalanceChecks): fully implement Safety and Balance chec…

### PR DESCRIPTION
**Pull request by AndyPerryKainos**
Fully implement Safety and Balance checks for Cat A Mod 2

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
IMG_0005
IMG_0006
IMG_0007
IMG_0008
IMG_0009

### CODE REVIEW

**Review by faggc — APPROVED** [review]

File: src/shared/constants/balance-questions.cat-a-mod2.constants.ts [hunk 1]
```diff
@@ -1,23 +1,22 @@
 import { VehicleChecksQuestion } from '../../providers/question/vehicle-checks-question.model';
 
-// TODO - PREP-AMOD2: Add balance questions (MES-4670)
 export default [
   {
-    code: 'T01',
+    code: 'Q01',
```
Comment (TomBillingtonUK) [inline src/shared/constants/balance-questions.cat-a-mod2.constants.ts]: There have been discussions on other categories where we shouldn't have leading zeros in the questions so it should be Q1, Q2 ... Can you double check if we should or shouldn't here

**Review by TomBillingtonUK — APPROVED** [review]
Looks good, just one query on leading zeros

File: src/shared/constants/balance-questions.cat-a-mod2.constants.ts (hunk 1 above)
Comment (AndyPerryKainos) [reply src/shared/constants/balance-questions.cat-a-mod2.constants.ts]: Who is the best person to make this decision?

**Comment by AndyPerryKainos** [comment]
Clarification from Matt Murray:
- Leading zeros removed
- Balance question code prefix is "BQ"
- Safety question code prefix is "SQ"

**Pull request merged**